lib/Manager.php

```diff
@@ -263,4 +329,55 @@ public function deleteMessagesForSessionIds($sessionIds) {
 			->orWhere($query->expr()->in('sender', $query->createNamedParameter($sessionIds, IQueryBuilder::PARAM_STR_ARRAY)));
 		$query->execute();
 	}
+
+	/**
+	 * @return string
+	 */
+	protected function getNewToken() {
+		$chars = str_replace(['l', '0', '1'], '', ISecureRandom::CHAR_LOWER . ISecureRandom::CHAR_DIGITS);
```

Comment:
@jancborchardt this is the token generation.
It uses only lowercase characters including 1 and l (lowercase L) as well as 0 (to avoid thinking of uppercase O)